examples/src/examples/compute/texture-gen/config.mjs

```diff
@@ -0,0 +1,32 @@
+/**
+ * @type {import('../../../../types.mjs').ExampleConfig}
+ */
+export default {
+    HIDDEN: true,
+    WEBGPU_REQUIRED: true,
+    FILES: {
+        'compute-shader.wgsl': `
```

Comment:
```suggestion
        'compute-shader.wgsl': /* wgsl */`
```

Reply:
any extension that supports it? I cannot find one.

Reply:
![image](https://github.com/playcanvas/engine/assets/48248865/3d5f929b-d8bb-4d75-976c-4f3a82fe0909)